From fc230d5ded73fe4049aa586eacbe4098aeff912f Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@paulbarker.me.uk>
Date: Wed, 12 Jun 2013 17:39:06 +0100
Subject: [PATCH] Fix warning with gcc 4.8.

The line "memset(pair, 0, sizeof(pair));" was causing the following warning:

	error: argument to 'sizeof' in 'memset' call is the same expression as
	the destination; did you mean to dereference

Compiling with -Werror (as specified in configure.ac) was causing this warning
to become an error. The line itself is not needed as memory is allocated via
calloc which performs zero initialization. Therefore, the line is removed to fix
the warning and does not need replacing anything.
---
 src/racoon/ipsec_doi.c | 1 -
 1 file changed, 1 deletion(-)

Upstream-Status: Accepted
	(in CVS HEAD and 0_8-stable, expected to be present in next release)

diff --git a/src/racoon/ipsec_doi.c b/src/racoon/ipsec_doi.c
index 2b48d49..4e9d3a8 100644
--- a/src/racoon/ipsec_doi.c
+++ b/src/racoon/ipsec_doi.c
@@ -1183,7 +1183,6 @@ get_proppair_and_doi_sit(sa, mode, doitype, sittype)
 			"failed to get buffer.\n");
 		goto bad;
 	}
-	memset(pair, 0, sizeof(pair));
 
 	bp = (caddr_t)(sab + 1);
 	tlen = sa->l - sizeof(*sab);
-- 
1.8.3

